tests/test_bs.py

```diff
@@ -142,7 +142,7 @@ def query_message(indx, namespace, client_po_name, fields, findFails=False):
     print("====================================================================")
     if findFails:
         print("Looking for pods that didn't hit:")
-        podnames = [hit.kubernetes.pod_name for hit in hits]
+        podnames = set([hit.kubernetes.pod_name for hit in hits])
```

Comment:
not sure that this is what we want every time.. maybe it should be done per test.